### DISCUSSION TITLE
GUI Options bar layout full width

### DISCUSSION THREAD

**Discussion by nofx77** [orestbida/cookieconsent#577] [Q&A]
In the `gui_options` with layout set at `bar` why is there a max-width of 32 rem on the id : #c-inr ? is there a way to change this for full width without overriding the css?

Design wise it's not appealing : 

![Screenshot 2023-09-27 at 3 15 02 PM](https://github.com/orestbida/cookieconsent/assets/1398110/57b2c075-e7d7-47ba-8179-0731d87ca2d4)

Would it be nicer like this? 

![Screenshot 2023-09-27 at 3 17 31 PM](https://github.com/orestbida/cookieconsent/assets/1398110/c11449c1-0b03-4845-adc8-7da070ea86cf)

Thanks!

**Comment by orestbida** [comment]
@nofx77 currently there is no way to change the `max-width` prop. without css/js. I will add a css variable to make this easier.

**Reply by nofx77** [reply]
Great and thank you for this great plug-in
